util_codes/DipoleWidget1D.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
```

Comment:
if we want to keep this code, can you move it to the `em_examples` directory? This will make importing it easier (esp. from other locations, like when building em.geosci.xyz). Thanks!